manifests/init.pp

```diff
@@ -298,6 +298,21 @@
       servername               => $foreman::servername,
     }
 
+    if $pulp {
```

Comment:
This should actually be `if $pulp_master` as `$pulp` denotes an external proxy with content. And note that with a master Pulp is configured by Katello so its not clear to me if the parameters below would be available. Perhaps include on pulp would make them so?